libs/l10n/l10n.ts

```diff
@@ -0,0 +1,30 @@
+type Locale =
+  | "en-US"
+  | "es"
+  | "fr"
+  | "ja"
+  | "ko"
+  | "pt-BR"
+  | "ru"
+  | "zh-CN"
+  | "zh-TW";
+type TranslatedLocale = Exclude<Locale, "en-US">;
+type Strings = {
+  "en-US": string;
+} & Record<TranslatedLocale, string>;
+
+function localString(strings: Strings) {
+  return (locale: string) => strings[locale] ?? strings["en-US"];
+}
+
+export const ONLY_AVAILABLE_IN_ENGLISH = localString({
+  "en-US": "This page is currently only available in English.",
+  es: "Esta página está disponible solo en inglés.",
```

Comment:
@mdn/yari-content-es Can you please check this translation? 🙏

Reply:
It looks good to me :+1:  cc @Graywolf9